packages/angular_devkit/build_angular/test/browser/rebuild_spec_large.ts

```diff
@@ -267,7 +267,7 @@ describe('Browser Builder rebuilds', () => {
 
   it('rebuilds after errors in AOT', async () => {
     // DISABLED_FOR_IVY - These should pass but are currently not supported
-    if (ivyEnabled) {
+    if (!veEnabled) {
```

Comment:
General note: in the near future we should investigate why this test is still broken.